crates/rune/src/runtime/function.rs

```diff
@@ -1058,4 +1064,101 @@ mod tests {
         assert_send::<SyncFunction>();
         assert_sync::<SyncFunction>();
     }
+
+    #[derive(Default)]
+    struct MyAny {}
```

Comment:
Please move these tests into a dedicated file in the [`tests` folder](https://github.com/rune-rs/rune/tree/main/crates/rune/src/tests).

Reply:
Done, thanks!